2022-08-10 14:46:19,472 INFO    Thread-43 :599705 [wandb_setup.py:_flush():76] Configure stats pid to 599705
2022-08-10 14:46:19,473 INFO    Thread-43 :599705 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-10 14:46:19,473 INFO    Thread-43 :599705 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-10 14:46:19,473 INFO    Thread-43 :599705 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_c', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': 'tnx91nle', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-1rpldmq6/config-tnx91nle.yaml', 'sweep_id': '1rpldmq6'}
2022-08-10 14:46:19,473 WARNING Thread-43 :599705 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-10 14:46:19,473 INFO    Thread-43 :599705 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-10 14:46:19,473 INFO    Thread-43 :599705 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220810_144619-tnx91nle/logs/debug.log
2022-08-10 14:46:19,473 INFO    Thread-43 :599705 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220810_144619-tnx91nle/logs/debug-internal.log
2022-08-10 14:46:19,477 INFO    Thread-43 :599705 [wandb_init.py:init():495] calling init triggers
2022-08-10 14:46:19,477 INFO    Thread-43 :599705 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 32, 'IMS_PER_BATCH': 4, 'iteration': 5000, 'learning_rate': 0.00017589396337130392, 'model': 'COCO-Detection/faster_rcnn_R_101_FPN_3x.yaml'}
config: {}
2022-08-10 14:46:19,480 INFO    Thread-43 :599705 [wandb_init.py:init():548] starting backend
2022-08-10 14:46:19,481 INFO    Thread-43 :599705 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-10 14:46:19,491 INFO    Thread-43 :599705 [backend.py:ensure_launched():217] starting backend process...
2022-08-10 14:46:19,537 INFO    Thread-43 :599705 [backend.py:ensure_launched():222] started backend process with pid: 615897
2022-08-10 14:46:19,538 INFO    Thread-43 :599705 [wandb_init.py:init():558] backend started and connected
2022-08-10 14:46:19,552 INFO    Thread-43 :599705 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 32, 'IMS_PER_BATCH': 4, 'iteration': 5000, 'learning_rate': 0.00017589396337130392, 'model': 'COCO-Detection/faster_rcnn_R_101_FPN_3x.yaml'}
2022-08-10 14:46:19,556 INFO    Thread-43 :599705 [wandb_init.py:init():634] updated telemetry
2022-08-10 14:46:19,558 INFO    Thread-43 :599705 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-10 14:46:20,364 INFO    Thread-43 :599705 [wandb_run.py:_on_init():1919] communicating current version
2022-08-10 14:46:20,427 INFO    Thread-43 :599705 [wandb_run.py:_on_init():1923] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-10 14:46:20,427 INFO    Thread-43 :599705 [wandb_init.py:init():700] starting run threads in backend
2022-08-10 14:46:21,527 INFO    Thread-43 :599705 [wandb_run.py:_console_start():1893] atexit reg
2022-08-10 14:46:21,530 INFO    Thread-43 :599705 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-10 14:46:21,530 INFO    Thread-43 :599705 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-10 14:46:21,530 ERROR   Thread-43 :599705 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-10 14:46:21,530 INFO    Thread-43 :599705 [wandb_init.py:init():732] run started, returning control to user process
2022-08-10 14:46:22,285 INFO    Thread-43 :599705 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_c/tnx91nle
2022-08-10 14:46:22,285 INFO    Thread-43 :599705 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-10 14:46:22,286 INFO    Thread-43 :599705 [wandb_run.py:_restore():1834] restore
2022-08-10 14:46:22,289 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-10 14:46:22,700 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-10 14:46:23,386 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-10 14:46:23,488 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-10 14:46:23,591 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-10 14:46:23,693 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5951
}

2022-08-10 14:46:23,796 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3042
  total_bytes: 5951
}

2022-08-10 14:46:23,899 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5309
  total_bytes: 5951
}

2022-08-10 14:46:24,001 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5309
  total_bytes: 5951
}

2022-08-10 14:46:24,103 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5309
  total_bytes: 5951
}

2022-08-10 14:46:24,206 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-10 14:46:24,308 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-10 14:46:24,411 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-10 14:46:24,513 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-10 14:46:24,615 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-10 14:46:25,117 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}

2022-08-10 14:46:25,670 INFO    Thread-43 :599705 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5951
  total_bytes: 5951
}
local_info {
}

2022-08-10 14:46:26,808 INFO    Thread-43 :599705 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-10 14:46:26,808 INFO    Thread-43 :599705 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-10 14:46:26,813 INFO    Thread-43 :599705 [wandb_run.py:_footer_sync_info():3055] logging synced files
